stock_picking_ready_flag/views/stock_picking_views.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='scheduled_date']" position="before">
+                <field
+                    name="is_ready"
+                    attrs="{'invisible':[('picking_type_code','!=','outgoing')]}"
+                />
+            </xpath>
+        </field>
+    </record>
+    <record id="vpicktree" model="ir.ui.view">
```

Comment:
I know the odoo stock_picking_view define this id.
But I think "view_picking_tree" is much better.

You don't need to replace `ref="stock.vpicktree"`, just rename record id.

Reply:
Actually odoo/addons/views/stock_picking.xml defines original record of "vpicktree".
In my opinion, changing the record ID does not make much sense, rather forces us to adjust the priority of the view to be displayed.

Reply:
`id="vpicktree"` should be fine according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml.